apps/atomy-q/API/app/Http/Controllers/Api/V1/QuoteSubmissionController.php

```
@@ -202,6 +219,115 @@ public function updateStatus(QuoteSubmissionStatusRequest $request, string $id):
         ]);
     }
 
+    public function storeSourceLine(ManualNormalizationSourceLineRequest $request, string $id): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)